Makefile

```diff
@@ -1,41 +1,26 @@
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
+.PHONY: build release install uninstall clean test doc reindent
 
-SETUP = ocaml setup.ml
+build:
+	jbuilder build @install --dev
 
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+release:
+	jbuilder build @install
 
-doc: setup.data build
-	$(SETUP) -doc $(DOCFLAGS)
+install:
+	jbuilder install
 
-test: setup.data build
-	$(SETUP) -test $(TESTFLAGS)
-
-all:
-	$(SETUP) -all $(ALLFLAGS)
-
-install: setup.data
-	$(SETUP) -install $(INSTALLFLAGS)
-
-uninstall: setup.data
-	$(SETUP) -uninstall $(UNINSTALLFLAGS)
-
-reinstall: setup.data
-	$(SETUP) -reinstall $(REINSTALLFLAGS)
+uninstall:
+	jbuilder uninstall
 
 clean:
-	$(SETUP) -clean $(CLEANFLAGS)
-
-distclean:
-	$(SETUP) -distclean $(DISTCLEANFLAGS)
-
-setup.data:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
+	jbuilder clean
 
-configure:
-	$(SETUP) -configure $(CONFIGUREFLAGS)
+test:
+	jbuilder runtest
 
-.PHONY: build doc test all install uninstall reinstall clean distclean configure
+# requires odoc
+doc:
+	jbuilder build @doc
 
-# OASIS_STOP
+reindent:
+	ocp-indent --inplace **/*.ml*
```

Comment:
Ooops sorry, this was my recommendation.
Maybe we should split it into two commands in the future:
```
ocp-indent --inplace **/*.ml
ocp-indent --inplace **/*.mli
```